hail/src/main/scala/is/hail/asm4s/ClassBuilder.scala

```diff
@@ -234,15 +241,8 @@ trait WrappedClassBuilder[C] extends WrappedModuleBuilder {
   def newStaticMethod(name: String, parameterTypeInfo: IndexedSeq[TypeInfo[_]], returnTypeInfo: TypeInfo[_]): MethodBuilder[C] =
     cb.newStaticMethod(name, parameterTypeInfo, returnTypeInfo)
 
-  def getOrGenMethod(
-    baseName: String, key: Any, argsInfo: IndexedSeq[TypeInfo[_]], returnInfo: TypeInfo[_]
-  )(body: MethodBuilder[C] => Unit): MethodBuilder[C] =
-    cb.getOrGenMethod(baseName, key, argsInfo, returnInfo)(body)
-
```

Comment:
This was a while ago...
Indeed, this wasn't used and I think the problem was in ownership. We had multiple method memos, one in EmitClassBuilder and one in ClassBuilder.
The reason the memo doesn't exist in ClassBuilder is because there would be multiple methods with the same signature (owing to type erasure).
I made the decision to move this to the highest level, therefore.

Really, I'd prefer if we organised our code a bit better to define and then use later (rather than define inline). That's a much larger change, however.